ksql-parser/src/main/java/io/confluent/ksql/parser/rewrite/StatementRewriter.java

```diff
@@ -150,6 +151,22 @@ protected Query visitQuery(final Query node, final C context) {
       );
     }
 
+    @Override
+    protected AstNode visitExplain(final Explain node, final C context) {
+      if (!node.getStatement().isPresent()) {
+        return node;
+      }
+
+      final Statement original = node.getStatement().get();
+      final Statement rewritten = (Statement) rewriter.apply(original, context);
+
+      return new Explain(
+          node.getLocation(),
+          node.getQueryId(),
+          rewritten == null ? Optional.of(original) : Optional.of(rewritten)
```

Comment:
Why would this be null?

Reply:
If there's no rule/code to rewrite it, e.g. if it wasn't a query. It's a weird edge case...

Reply:
I've resolved this....